src/zpotrf_L.jdf

```diff
@@ -88,6 +112,7 @@ potrf_zpotrf(k) [high_priority = on flops = inline_c %{ return FLOPS_ZPOTRF( CLE
 
 // Execution space
 k = 0 .. descA->mt-1
+info = 0  /* For the info in the case of recursive calls */
```

Comment:
It is somewhat hackish to use the execution space to store body outputs. We could access the global INFO and perform a  local reduction with an atomic on it during `update_INFO` (it will do 1 more atomic per subtask, but probably not that performance critical).